treeherder/log_parser/tasks.py

```diff
@@ -142,29 +82,28 @@ def parse_job_log(project, func_name, routing_key, job_guid, job_log_id):
 @retryable_task(name='log-parser', max_retries=10)
 @taskset
 @parser_task
-def parse_log(project, job_guid, job_log, _priority):
+def parse_log(job_log, _priority):
     """
     Call ArtifactBuilderCollection on the given job.
     """
-    post_log_artifacts(project, job_guid, job_log)
+    post_log_artifacts(job_log)
 
 
 @retryable_task(name='store-failure-lines', max_retries=10)
 @taskset
 @parser_task
-def store_failure_lines(project, job_guid, job_log, priority):
+def store_failure_lines(job_log, priority):
     """This task is a wrapper for the store_failure_lines command."""
-    logger.debug('Running store_failure_lines for job %s' % job_guid)
-    failureline.store_failure_lines(project, job_guid, job_log)
+    logger.debug('Running store_failure_lines for job %s' % job_log.job.id)
+    failureline.store_failure_lines(job_log)
     if settings.AUTOCLASSIFY_JOBS:
-        autoclassify.apply_async(args=[project, job_guid],
+        autoclassify.apply_async(args=[job_id.job.id],
                                  routing_key="autoclassify.%s" % priority)
 
 
 @retryable_task(name='crossreference-error-lines', max_retries=10)
-def crossreference_error_lines(project, job_guid):
+def crossreference_error_lines(job_id):
     """This task is a wrapper for the crossreference error lines command."""
```

Comment:
This comment is out of date.

Also, looks like the rest of the changes to this function are incomplete? (Nothing has replaced the `call_command` yet)